lib/ProductOpener/Display.pm

```diff
@@ -7530,59 +7530,33 @@ sub display_product_search_or_add($)
 
 	my $title = lang("add_product");
 
-	my $or = $Lang{or}{$lc};
-	$or =~ s/( |\&nbsp;)?://;
-
 	my $html = '';
+	my $template_data_ref_content = {};
+	my $template_data_ref_content_producer = {};
+	$template_data_ref_content->{server_options_producers_platform} = $server_options{producers_platform};
 
 	# Producers platform: display an addition import products block
 
 	if ($server_options{producers_platform}) {
 
-		$html = <<HTML
-&rarr; <a href="/cgi/import_file_upload.pl">$Lang{import_product_data}{$lc}</a><br>
-&rarr; <a href="/cgi/import_photos_upload.pl">$Lang{import_product_photos}{$lc}</a><br>
-&rarr; <a href="/cgi/export_products.pl">$Lang{export_product_data_photos}{$lc}</a><br>
-&rarr; <a href="/cgi/remove_products.pl">$Lang{remove_products_from_producers_platform}{$lc}</a><br>
-</p>
-HTML
-;
+		process_template('web/common/includes/display_product_search_or_add_producer.tt.html', $template_data_ref_content_producer, \$html) || ($html = "template error: " . $tt->error());
```

Comment:
here we use $html for 2 different content, we should use a new variable $html_producer instead, and declare it inside this block.
The declaration of my $template_data_ref_content_producer = {}; should be moved inside this block as well.